packages/gatsby-theme-newrelic/src/components/Button.js

```diff
@@ -65,12 +69,14 @@ const Button = styled.button`
   border-style: solid;
   ${({ variant }) => styles.variant[variant]}
   ${({ size }) => styles.size[size]}
+  ${({ disabled }) => disabled === true && styles.disabled}
```

Comment:
```suggestion
  ${({ disabled }) => disabled && styles.disabled}
```

Is there a reason you need to compare with `=== true` instead of just using `disabled &&` since you've specified that `disabled` is a boolean?

Reply:
I originally had that, but it was always making it `disabled`. Not sure why and my brain wasn't working. I can take an additional look at it.

Reply:
Huh thats odd. `false && styles.disabled` should return `false`, which would just get omitted. Maybe the `disabled `value` is set to something other than `false` in the "falsey" cases?